packages/default/scss/pivotgrid/_theme.scss

```diff
@@ -1,5 +1,170 @@
 @include exports("pivotgrid/theme") {
 
+    .k-pivotgrid,
+    .k-pivotgrid-values {
+        @include fill (
+            $pivotgrid-text,
+            $pivotgrid-bg,
+            $pivotgrid-border
+        );
+    }
+
+    .k-pivotgrid-empty-cell,
+    .k-pivotgrid-column-headers,
+    .k-pivotgrid-row-headers {
+        @include fill (
+            $color: $pivotgrid-headers-text,
+            $bg: $pivotgrid-headers-bg
+        );
+    }
+
+    .k-pivotgrid-column-headers,
+    .k-pivotgrid-row-headers {
+        @include fill (
+            $border: $pivotgrid-headers-border
+        );
+    }
```

Comment:
consider combining the two selectors. you are setting empty cell border in the next line, so it will override the previous setting